File: python/4344_average.py
#BOJ 4344

C = int(input())

for _ in range(C):
    lst = list(map(int, input().split()))
    N = lst.pop(0)
    count = 0
    average = sum(lst) / N
    for i in lst:
        if i > average:
            count += 1
    print(f"%.3f%%" %(count / N * 100)) #"%%" is needed to print "%" in prinf-formatting

File: c/4344_average.c
//4344 B1 average

#include <stdio.h>

int main()
{
    int C;
    scanf("%d", &C);

    for (int i = 0; i < C; i++)
    {
        int N;
        double sum = 0, count = 0, average;
        scanf("%d", &N);
        int arr[N];
        for (int j = 0; j < N; j++)
        {
            scanf("%d", &arr[j]); 
        }

        for (int k = 0; k < N; k++)
        {
            sum += arr[k];
        }
        average = sum / N;

        for (int l = 0; l < N; l++)
        {
            if (arr[l] > average)
                count += 1;
        }
        printf("%.3f%%\n", count / N * 100);
    }
}